api/services/images/v1/images.proto

```diff
@@ -115,4 +115,8 @@ message ListImagesResponse {
 
 message DeleteImageRequest {
 	string name = 1;
+
+	// Sync indicates that the delete and cleanup should be done
+	// synchronously before returning to the caller
+	bool sync = 2;
```

Comment:
Let's make it clear here that the default is `false` and make that an API guarantee. Only thing needed to assure that is to document it here: the code in place follows that sufficiently.